frontend/src/components/ViewActivities/activity.js

```diff
@@ -54,7 +54,9 @@ class Activity extends React.Component {
         </Card.Body>
       );
     } else { // Edit mode.
-      return ( <EditActivity activity={this.props.activity} submitFunction={this.finishEditActivity} />);
+      return ( <EditActivity activity={this.props.activity} 
+        submitFunction={this.finishEditActivity} 
+        new={false} />);
```

Comment:
I think it's good to have each attribute spaced out in this way when there's too many for one line:
```suggestion
      return ( 
          <EditActivity
              activity={this.props.activity}
              submitFunction={this.finishEditActivity}
              new={false}
          />
      );
```